src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerPanel.java

```diff
@@ -629,11 +649,11 @@ private void initializeHiddenEdges() {
 
 		// only if we don't allow sources and targets internal to paths
 		if (!_allowSourcesTargetsInPaths) {
-			// hides all incoming edges to source nodes
+			// hides all incoming directed edges to source nodes
```

Comment:
Is it correct to hide only incoming directed edges? Isn't it important also to hide the "version" of the undirected edges that are incoming? For instance, suppose you have an undirected edge (a, b). You use a->b in one path and b->a in another path. At this stage, shouldn't the edge be hidden in both directions?